src/run-cache.ts

```
@@ -94,6 +140,132 @@ class RunCache {
+      return;
+    }
+
+    // Choose eviction strategy based on configuration
+    switch (RunCache.config.evictionPolicy) {
+      case EvictionPolicy.LRU:
+        RunCache.evictLRU(entriesToEvict);
+        break;
+      case EvictionPolicy.LFU:
+        RunCache.evictLFU(entriesToEvict);
+        break;
+      case EvictionPolicy.NONE:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)